### ISSUE TITLE
Failed to generate auth token

### ISSUE THREAD

**Issue by ThatDumbUser** [PalatinCoder/MMM-GoogleBirthdaysProvider#3]
Having a problem trying to generate the auth token. This was from the log file:

0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'token:generate' ]
2 info using npm@6.4.1
3 info using node@v10.15.3
4 verbose run-script [ 'pretoken:generate', 'token:generate', 'posttoken:generate' ]
5 info lifecycle mmm-googlebirthdaysprovider@0.2.0~pretoken:generate: mmm-googlebirthdaysprovider@0.2.0
6 info lifecycle mmm-googlebirthdaysprovider@0.2.0~token:generate: mmm-googlebirthdaysprovider@0.2.0
7 verbose lifecycle mmm-googlebirthdaysprovider@0.2.0~token:generate: unsafe-perm in lifecycle true
8 verbose lifecycle mmm-googlebirthdaysprovider@0.2.0~token:generate: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/pi/MagicMirror/modules/MMM-GoogleBirthdaysProvider/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games
9 verbose lifecycle mmm-googlebirthdaysprovider@0.2.0~token:generate: CWD: /home/pi/MagicMirror/modules/MMM-GoogleBirthdaysProvider
10 silly lifecycle mmm-googlebirthdaysprovider@0.2.0~token:generate: Args: [ '-c', 'node ./google-api-credentials/generate-auth-token.js' ]
11 silly lifecycle mmm-googlebirthdaysprovider@0.2.0~token:generate: Returned: code: 1  signal: null
12 info lifecycle mmm-googlebirthdaysprovider@0.2.0~token:generate: Failed to exec token:generate script
13 verbose stack Error: mmm-googlebirthdaysprovider@0.2.0 token:generate: `node ./google-api-credentials/generate-auth-token.js`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
13 verbose stack     at EventEmitter.emit (events.js:189:13)
13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:189:13)
13 verbose stack     at maybeClose (internal/child_process.js:970:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
14 verbose pkgid mmm-googlebirthdaysprovider@0.2.0
15 verbose cwd /home/pi/MagicMirror/modules/MMM-GoogleBirthdaysProvider/google-api-credentials
16 verbose Linux 4.19.30-v7+
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "token:generate"
18 verbose node v10.15.3
19 verbose npm  v6.4.1
20 error code ELIFECYCLE
21 error errno 1
22 error mmm-googlebirthdaysprovider@0.2.0 token:generate: `node ./google-api-credentials/generate-auth-token.js`
22 error Exit status 1
23 error Failed at the mmm-googlebirthdaysprovider@0.2.0 token:generate script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]

**Comment by PalatinCoder** [comment]
I can't get any knowledge out of this, other than that `generate-auth-token.js` ended unexpectedly, but not why.
Is there some more output or more information that you could provide?

**Comment by ThatDumbUser** [comment]
I will get what ever information you need to help debug this, but bare with me I am a new person trying this Linux/Pi stuff out for the first time hence the username. Is there a flag I can append to the execute line to output more debug information for you?

**Comment by PalatinCoder** [comment]
Don't worry, everyone has to start somewhere :)
There is no switch for additional logging, however there should be more output than that.
Could you provide the full output from the terminal? (Including the command, basically everything between the prompts :D)

**Comment by PalatinCoder** [comment]
Closed due to inactivity. Feel free to reopen if needed.

**Comment by lacalos** [comment]
i am unable to create and generate the token api. I put the photo of how I made the "generate-auth-token.js" file. I add that I downloaded an api from google cloud by creating a web application api and a desktop one. the new api wants more information and I don't understand what to put on. I downloaded the web application API and I did the procedures but it gives me this screen. hope you have time to update and watch this comment. thanks
![IMG_20210603_033649](https://user-images.githubusercontent.com/51279218/120574264-46747680-c41f-11eb-8240-a44cf1ff99cf.jpg)
